apps/src/lib/kits/maker/portScanning.js

```diff
@@ -30,9 +31,10 @@ export function findPortWithViableDevice() {
       .then(list => {
         const bestOption = getPreferredPort(list);
         if (bestOption) {
-          return bestOption.comName;
+          return Promise.resolve(bestOption.comName);
```

Comment:
You shouldn't need to wrap this. From the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then):
> When a value is simply returned from within a then lambda, it will effectively return `Promise.resolve(<value returned by whichever handler was called>)`.